GHzDACs/ghz_fpga_server.py

```diff
@@ -880,7 +880,7 @@ class FPGAServer(DeviceServer):
     """Server for GHz DAC and ADC boards.
     """
     name = 'GHz FPGAs'
-    retries = 5
+    retries = 100
```

Comment:
I don't know. I want to make this number large and improve the logging so we can find out what's going on.